index.html

```diff
@@ -26,44 +26,57 @@ <h2>Make a mistake and it's game over!</h2>
         </div>
     </div>
 </div>
+<div id="gameOverModal" class="modal">
+    <div class="modal-content">
+        <div class="gameOverTop">
+            <div class="leftX"></div>
+            <h1>Game Over :(</h1>
+            <span class="gameOverClose close">X</span>
+        </div>
+        <div class="gameOverBtns">
+            <button class="replayButton">Play Again</button>
+        </div>
+    </div>
+</div>
 <div class = 'titles'>
     <div class="gameTitle">Where's the dog been?</div>
     <div class="levelNum">Level X</div>
 </div>
 <div class = 'gameBoxes'>
     <div class = 'row'>
```

Comment:
code consistency, why do these have spaces around the `=`? `class = '` dont do this